.devcontainer/scripts/2-updateContent.ps1

```diff
@@ -0,0 +1,28 @@
+#Docs: https://containers.dev/implementors/json_reference/#lifecycle-scripts
+. $PSScriptRoot/shared.ps1
+
+# Fetch upstream tags. The PowerShell dotnet restore requires this (GetPSCoreVersionFromGit)
+foreach ($source in 'upstream', 'origin') {
+    $remotes = git remote
+    if ($remotes -contains $source) {
+        Write-Host -Foreground Cyan "Fetching $source tags"
+        git fetch $source --tags
+    }
+}
+
+Import-Module ./build.psm1
+log 'Bootstrap PowerShell Prerequisites'
+& sudo pwsh -c {
+    Import-Module ./build.psm1
```

Comment:
Should we add `-Force`?

Reply:
This only happens on container startup so nothing is loaded at this point anyways. We can but it won't make a difference.